src/odemis/driver/test/orsay_test.py

```diff
@@ -1930,11 +1954,47 @@ def test_shift(self):
         rng_shift = self.scanner.shift.range
         self.scanner.shift.value = rng_shift[1]
         sleep(1)
-        self.assertEqual(self.scanner.shift.value, rng_shift[1])
         self.assertNotEqual(self.scanner.shift.value, (0, 0))
 
         self.scanner.shift.value = (0, 0)
 
+    def test_shift_calib(self):
+        """
+        Check that the shift is independent from the probe current, based on the
+        "base shift", stored in MD_SCAN_OFFSET_CALIB.
+        """
+        self.scanner.horizontalFoV.value = 100e-6  # m, a little large FoV, for the large shifts to work
+
+        base_shifts = self.scanner.getMetadata()[model.MD_SCAN_OFFSET_CALIB]
+        logging.info("MD_SCAN_OFFSET_CALIB = %s", base_shifts)
+        # There is one more choice in the currents that the base shift, which is "None"
+        self.assertEqual(set(base_shifts.keys()), set(self.scanner.probeCurrent.choices) - {None})
+
+        orig_shift = self.scanner.shift.value
+
+        # Changing current doesn't change the shift (but it may do that on fib_beam)
```

Comment:
It is bit unclear for me, does the comment wants to say the following - when the current change of the beam is requested -> only the current changes and the shift of the fib beam remains unchanged?

Reply:
Yes, that's correct. The shit.value remains the same, and the center of the FoV remains the same (assuming the calibration is good!).